From: Agathe Porte <agathe.porte@canonical.com>
Date: Tue, 2 Jan 2024 15:37:48 +0100
Subject: spinfinity: do not rely on meson for symlink creation

Meson is currently not able to create broken symlinks, which is required
when packaging using a PKGBUILDDIR. This is because the created symlink
will point to a file in /usr/share/plymouth/... but the file will not
be installed yet in the host file system but only in the prefix (and after
the meson install anyways).

Upstream issue: https://github.com/mesonbuild/meson/issues/12253

Drop the meson link creation and rely on debian/rules for copying the logo
for now.

Forwarded: not-needed
---
 themes/spinfinity/meson.build | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/themes/spinfinity/meson.build b/themes/spinfinity/meson.build
index f48e8e5..5a2050c 100644
--- a/themes/spinfinity/meson.build
+++ b/themes/spinfinity/meson.build
@@ -53,8 +53,3 @@ install_data(
   'throbber-33.png',
   install_dir: plymouth_theme_path / 'spinfinity',
 )
-
-install_symlink('header-image.png',
-  install_dir: plymouth_theme_path / 'spinfinity',
-  pointing_to: plymouth_logo_file,
-)
